eng/restore/repoRestore.targets

```diff
@@ -1,12 +1,23 @@
-<Project InitialTargets="_ClearResolvePackageAssets">
+<Project InitialTargets="_ClearResolvePackageAssets" TreatAsLocalProperty="ExcludeRestorePackageImports">
```

Comment:
Can you please explain why this is needed as part of this change?

Reply:
nuget prevents importing props and targets when calling the restore target since it could make a second restore have a different result in case these props and targets added a PackageReference. There is a comment I added in the targets that explain this further. As to why do we need it in this change, is because we use a Packaging library task `GetLastStablePackage` which needs for `$(PackagingTaskDir)` to be set, and it will only be set if we import `Packaging.common.targets` which not happen unless we special case Tools.proj.nuget.g.targets and do import it even in restore.

Reply:
Have you thought about converting the packaging arcade package to an msbuild sdk instead?